### DISCUSSION TITLE
Quiz: 5.3 Computing Metrics Efficiently with TorchMetrics (Part 2) - Error in formulation

### DISCUSSION THREAD

**Discussion by ghost** [Lightning-AI/dl-fundamentals#83] [Lecture videos and quizzes] [answered]
Hi,

I believe this should be change from:
Which of the following is **not a correct way** of computing the validation set accuracy in the validation_step method?
to: 
Which of the following is **a correct way** of computing the validation set accuracy in the validation_step method?

If not, please review the question and answer. It's not clear to me.

Thank you

**Comment by harishb00** [comment]
@rasbt 
![image](https://github.com/Lightning-AI/dl-fundamentals/assets/43300971/600c51d5-6991-4eb6-b543-19946ad7f551)

The question is mentioning **not** which means _**choose the wrong one from the choices**_. Even after choosing the wrong one, it throws error. Either question should be reformulated or the answer. Please correct me if I'm wrong.

Thanks.

**Reply by rasbt** [reply, marked as answer]
Thanks for pointing this out, this was indeed flipped. I just changed the wording to "Which of the following are correct ways of computing the validation set accuracy in the validation_step method?" to fix this. Thanks!!
